packages/react-strict-dom/src/native/stylex/index.js

```diff
@@ -664,6 +664,26 @@ export function props(
           );
         }
       }
+      // caretColor polyfill
+      else if (styleProp === 'caretColor') {
+        nativeProps.cursorColor = styleValue;
```

Comment:
I moved this to a final `else` clause at the bottom, which simplified the code slights and avoided a cursorColor being set when the value is transparent.